examples/problems/1000_compute_feature_correlation.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>What does this mean? do we know anything about these genes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>